lib/puppet/provider/service/systemd.rb

```diff
@@ -46,11 +67,20 @@ def status
   end
 
   def enable
+    output = systemctl("unmask", @resource[:name])
```

Comment:
Might doing a 'unmask' here defeat someone's explicit intention that the service be left un-enable-able?